src/alert.js

```diff
@@ -1,25 +1,25 @@
 import React, { forwardRef } from 'react'
 import PropTypes from 'prop-types'
+
+import { makeStyles } from '@material-ui/core/styles'
 import MUIAlert from '@material-ui/lab/Alert'
 import AlertTitle from '@material-ui/lab/AlertTitle'
 
-const Alert = forwardRef(({ severity, message, header, width, ...props }, ref) => {
-  const dimensions = { height: 'auto', width }
+const useStyles = makeStyles({ root: ({ width = '100%' }) => ({ height: 'auto', width }) })
+
+const Alert = forwardRef(({ message, header, width, ...props }, ref) => {
```

Comment:
I think width could be included as an inline styles attribute or a class passed down to the actual MUI component.

Reply:
class is fair, inline styles is what we're avoiding.